Gems/Atom/Feature/Common/Code/Source/ReflectionProbe/ReflectionProbe.cpp

```diff
@@ -120,15 +120,17 @@ namespace AZ
                     m_scene->RemoveRenderPipeline(m_environmentCubeMapPipelineId);
                     m_environmentCubeMapPass = nullptr;
 
-                    // restore exposure
-                    sceneSrg->SetConstant(m_iblExposureConstantIndex, m_previousExposure);
+                    // restore exposures
+                    sceneSrg->SetConstant(m_globalIblExposureConstantIndex, m_previousGlobalIblExposure);
+                    sceneSrg->SetConstant(m_skyBoxExposureConstantIndex, m_previousSkyBoxExposure);
 
                     m_buildingCubeMap = false;
                 }
                 else
                 {
-                    // set exposure to 0.0 while baking the cubemap
-                    sceneSrg->SetConstant(m_iblExposureConstantIndex, 0.0f);
+                    // set exposures to the user specified value while baking the cubemap
+                    sceneSrg->SetConstant(m_globalIblExposureConstantIndex, m_bakeExposure);
+                    sceneSrg->SetConstant(m_skyBoxExposureConstantIndex, m_bakeExposure);
```

Comment:
Will using the same m_bakeExposure for both IBL and skybox be appropriate in cases where users have tuned these to normally have different exposures? 

(I'm not sure exactly what use cases are being addressed by these changes, it would be helpful to have that info in the description).

Reply:
Currently I'm using the same exposure to generate a neutral cubemap, but this is an area that needs to be investigated further.